code/modules/mob/living/carbon/examine.dm

```diff
@@ -91,14 +91,15 @@
 	if(HAS_TRAIT(src, TRAIT_DUMB))
 		msg += "[t_He] seem[p_s()] to be clumsy and unable to think.\n"
 
-	if(fire_stacks > 0)
-		msg += "[t_He] [t_is] covered in something flammable.\n"
-	if(fire_stacks < 0 && fire_stacks > -1)
-		msg += "[t_He] look[p_s()] a little damp.\n"
-	if(fire_stacks <= -1 && fire_stacks > -5)
-		msg += "[t_He] look[p_s()] a little soaked.\n"
-	if(fire_stacks <= -5)
-		msg += "[t_He] look[p_s()] drenched.\n"
+	switch(fire_stacks)
```

Comment:
```
		if(1 to INFINITY)
			msg += "[t_He] [t_is] covered in something flammable.\n"
		if(-1)
			msg += "[t_He] look[p_s()] a little damp.\n"
		if(-2 to -4)
			msg += "[t_He] look[p_s()] a little soaked.\n"
		if(-5 to -INFINITY)
			msg += "[t_He] look[p_s()] drenched.\n"
```

1 to 2
2 to 6
isnt defined behavior and is listed in the ref so dont do that do this instead
1 to 2
3 to 6

Reply:
thinking about it, i'll probably just split this into it's own PR when i get home, since this should probably be in regardless as to whether or not the rework does